lifecycle/support/infrastructure-backups-and-restoration-policy.md

```diff
@@ -1,3 +1,122 @@
+ - 3.3 Tests are made once per year per application.
+ - 3.4 Testing is performed by a Support team engineer following established procedures, and includes the restoration of data from backups to a test environment to ensure that all critical data and applications can be restored successfully.
+ - 3.5 We maintain a backup testing log that tracks the status of all backup tests, including the date and time of the test, the results of the test, and any issues or errors encountered during the test. (To implement at a later date, for future review).
+ - 3.6 Backup testing logs are reviewed regularly to ensure that all tests are being completed successfully and to identify any issues that may require remediation.(To implement at a later date, for future review).
+ - 3.7 Our backup testing policy is reviewed regularly to ensure that it remains up-to-date and effective in ensuring the reliability and accuracy of our backups.
+
+## 4. Backup Integrity Verification
+ - 4.1 Annual integrity checks are performed to ensure that backups are free from corruption and can be relied upon in the event of a disaster or other issue that requires data restoration.
+ - 4.2 Integrity checks are performed on all Web application backups we store, including those stored locally and those stored off-site, to ensure that all copies of data are accurate and complete.
+ - 4.3 Testing the Integrity of an EC2 Snapshot:
+  - 4.3.1 Log in to the AWS Management Console.
+  - 4.3.2 Navigate to the EC2 Dashboard.
+  - 4.3.3 Select the snapshot you want to test.
+  - 4.3.4 Click on the "Actions" button and select "Create Volume".
+  - 4.3.5 Enter the desired configuration settings, such as the "Volume Type" and "Size".
+  - 4.3.6 Select the appropriate availability zone for the volume.
+  - 4.3.7 Click on the "Create Volume" button to start the volume creation process.
+  - 4.3.8 Once the volume has been created, select it in the Volumes section of the EC2 Dashboard.
+  - 4.3.9 Click on the "Actions" button and select "Attach Volume".
+  - 4.3.10 Select the appropriate instance and device for the volume.
+  - 4.3.11 Connect to the instance and verify that the volume is properly attached and functioning.
+  - 4.3.12 Verify the integrity of the data on the volume, such as by running file system checks or comparing it to the original instance.
+  - 4.3.13 If the data on the volume is consistent with the original instance, the integrity of the snapshot is confirmed.
+Detach and delete the volume once testing is complete.
+  - 4.3.14 Regularly test the integrity of your EC2 snapshots to ensure that they are properly backing up your data and applications.
+  - 4.3.1Note that the exact process may vary depending on the AWS region, instance type, and configuration settings. It's important to review the AWS documentation and best practices for EC2 snapshots to ensure that you are properly testing and verifying the integrity of your snapshots.
+
+## 5. Automating Backups

```

Comment:
This is a manual process that we could partially automate using the AWS Server Tracker, it's a little bit of work but doable. The reviewing of the instances is a lot more suited to being a manual process as it will require on the fly understanding and appreciation of what's working and what's not.

Ideally we should keep a record of these checks as well somewhere.